modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/sign-in/index.css

```diff
@@ -0,0 +1,107 @@
+.raylife-sign-in {
```

Comment:
Adjust all css to be able to use as much dialect and bootstrap as possible

Reply:
Some elements can't use dialect classes, because they are not editable